.github/workflows/conformance_with_build.yml

```diff
@@ -35,22 +35,22 @@ jobs:
             api-gateway-image: "consul-api-gateway:local-build"
             consul-image: "hashicorp/consul:1.11"
             envoy-image: "envoyproxy/envoy:v1.20-latest"
-            consul-k8s-version: "v0.48.0"
+            consul-k8s-version: "release/0.49.x"
```

Comment:
Could we wait til the upcoming consul-k8s release ~Thursday to keep this pinned to a released version? IIRC switching to an unreleased version may additionally require specifying an additional `imageK8s` config, see https://github.com/hashicorp/consul-api-gateway/pull/196

Reply:
I'm tinkering a little bit atm to determine why conformance is failing on the new control plane image